BTree.cpp

```diff
@@ -0,0 +1,184 @@
+
```

Comment:
There are 2 files with the same code, delete one of them.

Reply:
@somya-kapoor yes maam , by tomorrow morning i will commit it with all required  changes.